[Docs-IS] Clarify Attribute Configurations Scope (#5554)
wso2/docs-is: en/includes/guides/users/attributes/manage-attributes.md

7. Under **Attribute Configurations**, use the table to configure how attributes are handled for each entity.

    !!! Important "Scope"

>         These settings affect **UI presentation and validation only** in WSO2-managed UIs (Administrator Console, End-User Profile (i.e. My Account), Self-Registration). They **do not** modify backend/API behavior.

Reviewer: ```suggestion
        These settings only control how the attributes behave in WSO2-managed UIs (Administrator Console, End-User Profile (i.e. My Account), Self-Registration). They **do not** affect backend or API validation.
```
